server/template.go

```diff
@@ -11,8 +11,10 @@ import (
 )
 
 const mdCommentRegexPattern string = `(<!--[\S\s]+?-->)`
+const gitHubRegexPattern string = `(^|[^_\x60[:alnum:]])(@[[:alnum:]](-?[[:alnum:]]+)*)`
 
 var mdCommentRegex = regexp.MustCompile(mdCommentRegexPattern)
+var gitHubRegex = regexp.MustCompile(gitHubRegexPattern)
```

Comment:
should we call this `gitHubUsernameRegex` for clarity?

Reply:
Done in https://github.com/mattermost/mattermost-plugin-github/pull/177/commits/5db457eae041ea83f8f438228b523fcd6a600cec.